public/restaurants.js

```diff
@@ -0,0 +1,158 @@
+
+let button = document.getElementById('button')
+let message = document.getElementById('feedback')
+
+// variables for storing comments in local storage
+
+let savedComments = []
+let storedNotes = []
+
+// function to add inputted comments to comment space and call function to store them in local storage
+
+button.addEventListener('click', function () {
+	let transferredComments = localStorage.getItem(id + '-notes') || []
+	if (typeof transferredComments === 'string') {
+		savedComments = transferredComments.split(',')
+		savedComments.push(message.value)
+		populateStorage(id)		
+	}
+
+	if (typeof transferredComments === 'object') {
+		savedComments.push(message.value)
+		populateStorage(id)
+	}
+	comments.innerHTML += `<li style='padding-top: 1vh; list-style-type: none'>"${message.value}"</li>`
+	message.value = ""
+})
+
+// function to store feedback 
+
+function populateStorage(pageId) {
+	localStorage.setItem(pageId + '-notes', savedComments.toString())
+}
+
+// function to repopulate comments section
+
+function retrieveSavedComments() {
+	let retrievedComments = localStorage.getItem(id + '-notes')
+	if (typeof retrievedComments === 'string') {		
+		let storedNotes = retrievedComments.split(',')
+		storedNotes.forEach((storedComment) => {
+			comments.innerHTML += `<li style='padding-top: 1vh; list-style-type: none'>"${storedComment}"</li>`
+		})
+	}
+}
+
+// function to retrieve restaurant info from my api
+
+async function getMoreRestoInfo() {
+
+	let restaurant = await fetch('/api/indexRestaurants.json')
+		.then((response) => {
+			return response.json()
+		}).then((jsonArr) => {
+			return jsonArr
+		})
+
+	// iterates over index file to find the specific restaurant
+
+	let resto = restaurant.find((obj) => obj.id === id)
+
+	// initializing variables for info to be added to sidebar on restaurant page
+
+	let name = resto.name
+	let address = resto.address
+	let phone = resto.phone
+	let website = resto.website
+	let hours = resto.hours
+	let notes = resto.notes
+
+	// checks for missing or aberrant information 
+
+	if (!resto.hasOwnProperty('hours')) { 
+		hours = 'No hours available'
+	}
+
+	if (!resto.hasOwnProperty('website')) { 
+		website = ''
+	}
+	if (phone.length <= 8) {
+		phone = "802-" + phone
+	}
+	// 
+	// adds restaurant info to restaurant page
+
+	restaurantInfo.innerHTML = `<li><p style='font-size: 3vh; text-decoration: underline'>${name}</p></li><li><p>${address}</p></li><li><p>${phone}</p></li><li><a href='${website}' target="_blank">${website}</a></li><li><p>${hours}</p></li>`
+
+	// iterates over notes array and adds each comment to the comments box
+
+	notes.forEach((note) => {
+		comments.innerHTML += `<li style='padding-top: 1vh; list-style-type: none'>"${note}"</li>`
+	})
+
+	// retrieves comments from local storage and adds them to the comments box
+
+	retrieveSavedComments()
+
+	// adds restaurant name to page title
+
+	let title = document.getElementById('title')
+
+	title.innerHTML = name
```

Comment:
If you are just adding text to an element, then you can use textContent. innerHTML is useful when you have to add a whole element to the inside of another element like on line 104, [here](https://medium.com/better-programming/whats-best-innertext-vs-innerhtml-vs-textcontent-903ebc43a3fc) is a short article about the differences between innerHTML, textContent, and innerText.